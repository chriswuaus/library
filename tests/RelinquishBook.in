relinquish 100000 111111
add member beans
relinquish 100000 90909000
add collection test.csv
relinquish 10001010 111111
relinquish 100000 69696969
relinquish 100000 111111
rent 100000 111111
rent 100000 111112
relinquish 100000 111111
relinquish 100000 111112
relinquish 100000 111112
exit
