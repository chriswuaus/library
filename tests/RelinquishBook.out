user: No members in system.

user: Success.

user: No books in system.

user: 17 books successfully added.

user: No such member in system.

user: No such book in system.

user: Unable to return book.

user: Success.

user: Success.

user: Success.

user: Success.

user: Unable to return book.

user: Ending Library process.
